.travis.yml

```diff
@@ -1,15 +1,29 @@
-language: c
-
-os:
-  - osx
+language: objective-c
 
 env:
-  - NODE_VERSION="v0.10"
-  - NODE_VERSION="v0.11"
-  - NODE_VERSION="iojs-v1.0"
+  global:
+    - secure: f4hgPmIW9fXvgomICcfe7vjJY3TFzALboVq1Nz9xrX1T25oIt7+rHd8CjPbBZvkSgSBY3zDItY5n47kAbWoFZGqmWF9K26WEnljPLab4QeLmy7GGZ7/jCTyashPucc34b5SC2SuXAMbDSSbEImguY27GQmeCnRqpDOS7kDVGZEU=
+    - secure: oSMNhXFS6Cb9GLlYaFhqABWDLYcagkz6KY5T6jFzJ6tCLhBSVbdSX9p0RhOBcIV7vZ2WW12Nbl9DTtUgccuQJWZaI504KBq2dD94R54h4hY5ZDymsRLqNZ71fD/gh3dhV9ukciHaPYPK2/N1OSxDSUAeBsZtPJxOprOH07iUNpQ=
+  matrix:
+    - NODE_VERSION="v0.10"
+    - NODE_VERSION="v0.11"
+    - NODE_VERSION="v0.12"
+    - NODE_VERSION="iojs-v1.0"
```

Comment:
Seems like we'd probably have to get more specific than that. And I'm a bit concerned that the list there can't keep up with the pace of io.js releases.

What happens if a user attempts to install using a platform that hasn't been pre-compiled? Does it automatically fall back to regular node-gyp?

I still need to try this out to see how users would experience it. Apologies that I haven't had time to do so yet.